imls-backend/db/schema.sql

```diff
@@ -66,14 +219,77 @@ CREATE VIEW api.presences AS
 
 
 --
--- Name: libraries; Type: TABLE; Schema: imlswifi; Owner: -
+-- Name: heartbeats; Type: TABLE; Schema: imlswifi; Owner: -
 --
 
-CREATE TABLE imlswifi.libraries (
-    fscs_id character varying(16) NOT NULL
+CREATE TABLE imlswifi.heartbeats (
+    heartbeat_id integer NOT NULL,
+    fscs_id character varying(16) NOT NULL,
+    sensor_id integer NOT NULL,
+    hourly_ping timestamp with time zone DEFAULT CURRENT_TIMESTAMP
```

Comment:
by naming the ping "hourly_ping," it suggests (in the name of the field) that the ping is hourly. If we ever change our mind about the frequency, the field name becomes inaccurate. Should this be "last_ping" or "ping_time"? Or, something like that. ("last_ping" makes no sense, actually... so "ping_time" might?)

Reply:
Makes sense - I can update this on [a different PR for the heartbeat table](https://github.com/IMLS/estimating-wifi/pull/150) for cleanliness.